data/persons.txt

```diff
@@ -807,3 +807,82 @@ person "Subsidurial"
 		"never disabled"
 	system
 		government "Uninhabited"
+
+person "MCOfficer"
+	frequency 200
+	government "Author"
+	personality
+		disables heroic plunders opportunistic
+	phrase
+		phrase
+			"friendly author"
+	ship "Rais Iris XVIII" "MCOfficer"
```

Comment:
I would think that the model name should still be what the actual ship model is (Marauder Bactrian) while the ship name should be what MCO chose (Rais Iris XVIII), instead of having the ship model be MCO's ship name and the ship name being MCO. Tehhowch's ship for example isn't `ship "Power of the People" "tehhowch"`, it's `ship "Modified Osprey" "Power of the People"`.

Reply:
Sure, I can change that.

Reply:
Changed.